conf/docker.config

```diff
@@ -16,4 +16,8 @@ process {
         container = 'quay.io/biocontainers/multiqc:1.9--pyh9f0ad1d_0'
     }
 
+    withName:
+    'DOWNLOAD_GATK_JAR' {
+        container = 'alpine:3.14'
+    }
```

Comment:
@Mxrcon , perhaps we might not need to use an extra container? 

In these cases we can reuse (maybe?) the `mtbseq` container since its derived from `debian` and would be downloaded (and available) on the underlying machine anyhow.

What do you think?

Reply:
That's true, even alpine being a minimal container, we can use the mtbseq for it.